app/api/schema/custom_forms.py

```diff
@@ -30,6 +30,8 @@ class Meta:
     is_required = fields.Boolean(default=False)
     is_included = fields.Boolean(default=False)
     is_fixed = fields.Boolean(default=False)
+    is_custom_question = fields.Boolean(default=False, allow_none=False)
+    prompt = fields.Str(required=False, allow_none=True)
```

Comment:
@uds5501 are both attributes `required` and `allow_none` required here? I guess allow_none will be enough. What say?

Reply:
@shreyanshdwivedi  I think you are right. Will fix this.

Reply:
@shreyanshdwivedi take a look now?